templates/legal/developer-terms-and-conditions/2016-04-06.html

```diff
@@ -22,8 +22,7 @@ <h3 id="dev-terms-definitions">1. Definitions</h3>
         <li class="p-list__item">App(s): one or more applications or content items owned by you which you submit through the Developer Site and any associated screen shots and marketing materials provided by you.</li>
         <li class="p-list__item">Client Software: any utilities in Ubuntu, such as the Ubuntu Software Centre, which enable end users to discover, install, and remove software including Apps.</li>
         <li class="p-list__item">Developer Account: Your account on the Developer Site.</li>
-        <li class="p-list__item">Developer Site: Canonical&#39;s websites for uploading and managing Apps at <a href="https://myapps.developer.ubuntu.com/dev/click-apps/">myapps.developer.ubuntu.com/dev/click-apps</a></li>
-        <li class="p-list__item">Publishing Policy: the Developer Site policy for accepting Apps for publishing at developer.ubuntu.com, as updated from time to time.</li>
+        <li class="p-list__item">Developer Site: Canonical&#39;s websites for uploading and managing Apps at <a href="https://dashboard.snapcraft.io/snaps/">https://dashboard.snapcraft.io/snaps/</a>.</li>
```

Comment:
I'm not sure `Apps` (capitalised) is quite the right way to present this any more? But maybe I shouldn't nit-pick for now? Up to you.

Reply:
I asked KO to look, so will update with her feedback in another PR